test/electron-renderer/ui/HotkeyInput_spec.js

```diff
@@ -39,7 +39,7 @@ describe('<HotkeyInput />', () => {
 
   it('should render a text field with the correct default value', () => {
     const component = mount(<HotkeyInput label="Test Label" hotkeyAction="playPause" />, materialUIContext);
-    component.find(TextField).props().value.should.be.equal('Not Set');
+    component.find(TextField).props().value.should.be.equal(undefined);
```

Comment:
This is what is breaking the tests currently, it should be expecting "settings-option-hotkey-not-set"